inkscape/svg2shenzhen/export.py

```diff
@@ -730,8 +727,8 @@ def flatten_bezier(self):
             for node in layer.xpath(nodePath, namespaces=inkex.NSS):
                 if node.tag == inkex.addNS('path','svg'):
```

Comment:
addNS isn't reuquired any more. You can do `isinstance(node, inkex.PathElement)` there's also an iterator which can just give you all path elements without having to test for them (or use xpath directly)